helm-chart/renku-core/values.schema.json

```diff
@@ -0,0 +1,209 @@
+      },
+      "image": {
+        "description": "Docker image settings",
+        "properties": {
+            "repository": {
+                "type": "string"
+            },
+            "tag": {
+                "type": "string"
+            },
+            "pullPolicy": {
+                "type": "string"
+            }
+        },
+        "type": "object"
+      },
+      "imagePullSecrets": {
+        "description": "Secrets necessary for pulling the image",
+        "type": "array",
+        "items": {
+            "type": "object"
+        }
+      },
+      "nameOverride": {
+        "description": "Override for the chart name",
+        "type": "string"
+      },
+      "fullnameOverride": {
+        "description": "Override for the chart fullname",
+        "type": "string"
+      },
+      "service": {
+          "description": "Kubernetes service definition",
+          "properties": {
+            "type": {
+                "description": "The type of kubernetes service",
+                "type": "string"
+            },
+            "port": {
+                "description": "The port of the kubernetes service",
+                "type": "integer",
+                "minimum": 1
+            }
+          },
+          "type": "object"
+      },
+      "ingress": {
+        "description": "Kubernetes ingress definition",
+        "properties": {
+            "enabled": {
+                "description": "whether to enable the ingress or not",
+                "type": "boolean"
+            }
+        },
+        "type": "object"
+      },
+      "metrics": {
+          "description": "Definition of Redis Queue metrics",
+          "properties": {
+            "enabled":{
+                "description": "whether to enable redis queue metrics",
+                "type":"boolean"
+            },
+            "image": {
+                "description": "Image settings for rq metrics",
+                "properties": {
+                    "repository": {
+                        "type": "string"
+                    },
+                    "tag": {
+                        "type": "string"
+                    },
+                    "pullPolicy": {
+                        "type": "string"
+                    }
+                },
+                "type": "object"
+            }
+          },
+          "type": "object"
+      },
+      "resources": {
+          "description": "Kubenetes resource requests/limites (cpu/memory/etc.) to request",
+          "type": "object"
+      },
+      "nodeSelector": {
+          "description": "Node selector for deployment",
+          "type": "object"
+      },
+      "tolerations": {
+          "description": "Tolerations for deployment",
+          "type": "object"
+      },
+      "affinity": {
+          "description": "Affinity for deployment",
+          "type": "object"
+      },
+      "redis": {
+          "description": "Settings for the redis subchart",
+          "type": "object"
```

Comment:
redis properties are missing. Do we need them?

Reply:
They're for the redis subchart, so it doesn't make sense to validate values of another chart here.